docs/bundles/SyliusTaxationBundle/configuring_taxation.rst

```diff
@@ -45,52 +45,56 @@ Second thing to do is to classify the taxables, in our example we'll get two pro
 
     public function configureAction()
     {
-        $tshirt = // ...
-        $banana = // ... Some logic behind loading entities.
-
         $repository = $this->container->get('sylius.repository.tax_category');
 
-        $clothing = $repository->findOneBy(array('name' => 'Clothing'));
-        $food = $repository->findOneBy(array('name' => 'Food'));
+        $cloudServerTaxCategory = $repository->findOneBy(array('name' => 'Cloud Server'));
+        $sharedHostingTaxCategory = $repository->findOneBy(array('name' => 'Shared Hosting'));
+
+        $digitalOcean = new Server();
+        $taxCategory->setName('Digital Ocean');
+        $taxCategory->setTaxCategory($cloudServerTaxCategory);
+
+        $goDaddy = new Server();
+        $taxCategory->setName('GoDaddy');
+        $taxCategory->setTaxCategory($sharedHostingTaxCategory);
 
-        $tshirt->setTaxCategory($clothing);
-        $food->setTaxCategory($food);
+        $manager->persist($digitalOcean);
```

Comment:
If `Server` is a sylius resource it should also have a factory and then be added to database with `repository->add()`. Entity manager would not be needed